src/js/components/DataSearch/DataSearch.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import { Search } from 'grommet-icons/icons/Search';
+import { DataForm } from '../Data';
+import { FormContext } from '../Form/FormContext';
+import { TextInput } from '../TextInput';
+import { MessageContext } from '../../contexts/MessageContext';
+import { DataSearchPropTypes } from './propTypes';
+
+export const DataSearch = ({ ...rest }) => {
+  const { id: dataId, messages, noForm } = useContext(FormContext);
+  const { format } = useContext(MessageContext);
+
+  let content = (
+    <TextInput
```

Comment:
something to think about for future if we want to wrap this in `box` then be able to change the `width` I could see some maybe not wanting it to be the full width maybe size medium:
<img width="860" alt="Screen Shot 2022-12-16 at 2 06 03 PM" src="https://user-images.githubusercontent.com/42451602/208188909-8ef6f67b-49c5-496c-bc7c-c6f10d677109.png">